website/docs/deployment.mdx

```diff
@@ -54,6 +54,50 @@ Use [slorber/trailing-slash-guide](https://github.com/slorber/trailing-slash-gui
 
 :::
 
+## Choosing a hosting provider
+
+There are a few common hosting options:
+
+- [Self hosting](#self-hosting) with an HTTP server like Apache2 or Nginx;
+- Jamstack providers, e.g. [Netlify](#deploying-to-netlify) and [Vercel](#deploying-to-vercel). We will use Netlify as reference, but the same reasoning can apply to other providers.
```

Comment:
I like the Netlify DX but I don't think it's fair to use it as a primary example considering Vercel has 6000 free build minutes for hobby projects.

Reply:
I'm not familiar with Vercel and I'm slightly bugged by them commenting on commits (at least what I observed a lot with other projects; pretty sure it's configurable). Anyways I just changed it to `We will use them`